roles/grubcmdline/README.md

```diff
@@ -0,0 +1,51 @@
+stackhpc.grubcmdline
```

Comment:
Assume there are no substantial changes here apart from removing molecule tests?

Reply:
Yeah, literally copied the role in. Commit ID is in this commit message: https://github.com/stackhpc/ansible-collection-linux/pull/3/commits/f21e9dad6347f901e25123a51945ad8ef98979e1

Reply:
Oh yeah and updated the reference to grubcmdline to use the role internal to this collection: https://github.com/stackhpc/ansible-collection-linux/pull/3/commits/ca3f532db219e95c347065ff0fc5c81e39f5af66